cloud/aws-common/pom.xml

```diff
@@ -46,6 +46,10 @@
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-java-sdk-s3</artifactId>
         </dependency>
+        <dependency>
```

Comment:
let's add a comment to explain why this dependency is required, since we don't actually have any compile time dependency on it